matRad_calcCubes.m

```diff
@@ -107,8 +121,14 @@
     for f = 1:numel(dij.MC_tallies)
         tally = dij.MC_tallies{f};
         % skip tallies processed above
-        if ~isfield(resultGUI,tally)
-            resultGUI.(tally) = reshape(full(dij.(tally){scenNum}),dij.doseGrid.dimensions);
+        if ~isfield(resultGUI,tally) && ~contains(tally,'std')
+            tallyCut = strsplit(tally,'_');
+            if size(tallyCut,2) > 1
```

Comment:
This is now outdated since `matRad_calcParticleDoseMCtopas` and `matRad_calcParticleDoseMCsquare` handle this internally and all fields are processed in the new calcCubes function anyways, so there should not be any unprocessed fields in the first place.